### PR TITLE
Ea update azure ra

### PR DESCRIPTION
**Pull request by ml4**
## PR Objective

- [x] Fixing inaccurate docs
- [x] Making some docs easier to understand
- [ ] Adding/updating docs for new feature
- [ ] Changing behavior or layout of website

## Description

Amending Azure RA re https://app.asana.com/0/1168308898178155/1181442442841459
Add new diagram showing multi-AZ single region diagram for Azure.  Multi-region to be added in due course, not in this PR.

### CODE REVIEW

**Comment by straubt1** [comment]
Solid update, nice work here!

**Comment by ml4** [comment]
Several updates made based on CS team discussions - thanks Tom, Mike, Avocado Nathan and Dan!

File: content/source/docs/enterprise/before-installing/reference-architecture/azure.md [hunk 1]
```diff
@@ -86,8 +86,9 @@ An Azure Blob Storage
 must be specified during the Terraform Enterprise installation for application data to
 be stored securely and redundantly away from the Azure VMs running the
 Terraform Enterprise application. This Azure Blob Storage container must be in the same
-region as the VMs and Azure Database for PostgreSQL instance. It is recommended
-the virtual network containing the Terraform Enterprise servers be configured with a
+region as the VMs and Azure Database for PostgreSQL instance.
+
+It is recommended the virtual network containing the Terraform Enterprise servers be configured with a
```
Comment (nfagerlund) [inline content/source/docs/enterprise/before-installing/reference-architecture/azure.md:91]:
```suggestion
We recommend that the virtual network containing the Terraform Enterprise servers be configured with a
```

File: content/source/docs/enterprise/before-installing/reference-architecture/azure.md [hunk 2]
```diff
@@ -96,6 +97,8 @@ in the Azure Blob Storage container. This allows for further
 encryption](https://docs.microsoft.com/en-us/azure/storage/common/storage-service-encryption)
 by Azure Blob Storage if required by your security policy.
 
+Due to support within the product, and increased durability, for a single-region deployment, we recommend using zone-redundant storage (ZRS) which synchronously writes across three Azure availability zones in the region.  For a multi-region deployment, use geo-zone-redundant storage (GZRS) for added region redundancy.
```
Comment (nfagerlund) [inline content/source/docs/enterprise/before-installing/reference-architecture/azure.md:100]: What does "due to support within the product" mean?

Comment (ml4) [reply content/source/docs/enterprise/before-installing/reference-architecture/azure.md]: Clarified.

**Comment by nfagerlund** [comment]
Okay, something EXTREMELY bad has happened to this PR. I'm going to go in and rebase manually to try and undo whatever just happened here.

**Comment by nfagerlund** [comment]
I believe this repaired branch matches what you wanted to have happen: 

https://github.com/hashicorp/terraform-website/tree/ml4_azurera_fixup

If you can reset your PR branch to this one, I'll merge.

**Comment by nfagerlund** [comment]
superseded by #1411

**Pull request closed without merging**